ansible/playbook/roles/add_extra_users/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- set_fact:
+    path_users_pwd: "{{ openshift_env.config_dir }}/{{ users_pwd_file }}"
+
+- name: Add user/pwd to file
+  command: "htpasswd -b {{ path_users_pwd }} user{{ item }} pwd{{ item }}"
+  with_sequence: start={{ first_extra_user_offset }} count={{ number_of_extra_users }} format=%d
+
+- name: Restart master
```

Comment:
This step is not required. Can you remove it please ?

Reply:
Unless I am misunderstanding something, it is required. 
It adds the username / password for each user to be created into the master htpassword file.

Can you please clarify what you mean?

Reply:
Probably github is showing the line number incorrectly... Do you mean perhaps that restart is not needed?